Makefile

```diff
@@ -235,9 +242,10 @@ prove : $(tests) $(examples)
 # run prove target in parallel
 prv : ;@$(MAKE) $(MFLAGS) V=$(V) prove
 
-prove-allexamples : $(tests) $(examples) $(mfemexamples)
+alltests := $(tests) $(examples) $(mfemexamples) $(if $(PETSC_DIR),$(petscexamples))
```

Comment:
Do we want to add logic similar to your PETSc logic to check if MFEM is installed?

Reply:
Yeah, but not urgent.

Reply:
Added, but discovered problem with OCCA

Reply:
Namely, OCCA is having trouble finding the .okl